src/lib/reporters/Console.ts

```diff
@@ -22,6 +28,13 @@ export default class ConsoleReporter extends Reporter {
     this.console.error(this.formatError(error));
   }
 
+  @eventHandler()
+  log(data: any) {
+    if (this._debugLogs) {
```

Comment:
The reporter doesn't actually need to check any sort of debug condition. Other parts of Intern handle that (see `log` in `src/lib/executors/Executor.ts`), so the Console reporter should simply need to implement a handler for the `log` event.